test/50-assert-15-assert_output.bats

```diff
@@ -26,9 +26,19 @@ load test_helper
   [ "${lines[3]}" == '--' ]
 }
 
-@test 'assert_output(): reads <expected> from STDIN' {
+@test 'assert_output() - : reads <expected> from STDIN' {
```

Comment:
Please factor out common parts of tests to avoid duplicating code. Like it's done [here][option-test].

[option-test]: https://github.com/ztombol/bats-assert/blob/master/test/50-assert-15-assert_output.bats#L81-L94